internal/benchrunner/runners/rally/runner.go

```diff
@@ -692,7 +692,7 @@ func (r *runner) runGenerator(destDir string) error {
 
 	if r.options.RallyTrackOutputDir != "" {
 		r.persistRallyTrackHandler = func() error {
-			err := os.MkdirAll(r.options.RallyTrackOutputDir, os.ModeDir)
+			err := os.MkdirAll(r.options.RallyTrackOutputDir, os.ModePerm)
```

Comment:
`os.ModePerm` is 0777, do we need everyone to be able to make changes there? Wdyt about explicitly setting 0755?
```suggestion
			err := os.MkdirAll(r.options.RallyTrackOutputDir, 0755)
```